labs/ai-vision/analyze_video_workshop/notebooks/Offline_Video_Service.ipynb

```diff
@@ -80,7 +87,7 @@
    "outputs": [],
    "source": [
     "# Please use pip/pip3 as per availability\n",
-    "# !pip3 install --trusted-host=artifactory.oci.oraclecorp.com -i https://artifactory.oci.oraclecorp.com/api/pypi/global-dev-pypi/simple -U oci==2.115.2+preview.1.1678"
+    "# !pip3 install 2.115.2+preview.1.1678-py2.py3-none-any.whl"
```

Comment:
where is the wheel file ?

Reply:
We won't attach the wheel file right. User should request our team to get the wheel file. Isn't this the general process. Correct me if I am wrong

Reply:
pls confirm with ravi